packages/wp-story-editor/src/api/media.js

```diff
@@ -189,5 +185,7 @@ export function deleteMedia(config, mediaId) {
     path: addQueryArgs(`${config.api.media}${mediaId}/`, { _method: 'DELETE' }),
     data: { force: true },
     method: 'POST',
-  }).then(snakeToCamelCaseObjectKeys);
+  }).then((result) => {
+    return getResourceFromAttachment(result.previous);
```

Comment:
Similarly, the response is `deleteMedia` is not being used anywhere. I think we may remove the `.then` part.

https://github.com/google/web-stories-wp/blob/acb3df5148d37e8192023c70ea2d9256cf86241d/packages/story-editor/src/components/library/panes/media/local/deleteDialog.js#L62

Reply:
Yeah for this one it probably makes more sense to return void 👍